src/modules/registrypreview/RegistryPreviewUILib/RegistryValue.xaml.cs

```diff
@@ -46,12 +54,49 @@ public Uri ImageUri
             }
         }
 
-        public RegistryValue(string name, string type, string value)
+        public RegistryValue(string name, string type, string value, string key)
         {
             this.Name = name;
             this.Type = type;
             this.Value = value;
             this.ToolTipText = string.Empty;
+            this.Key = key;
+        }
+
+        // Commands
+        public ICommand CopyToClipboardEntry_Click => new RelayCommand(CopyToClipboardEntry);
+
+        public ICommand CopyToClipboardWithPath_Click => new RelayCommand(CopyToClipboardEntryWithPath);
+
+        public ICommand CopyToClipboardName_Click => new RelayCommand(CopyToClipboardName);
+
+        public ICommand CopyToClipboardType_Click => new RelayCommand(CopyToClipboardType);
+
+        public ICommand CopyToClipboardData_Click => new RelayCommand(CopyToClipboardData);
+
+        private void CopyToClipboardEntry()
+        {
+            ClipboardHelper.CopyToClipboardAction($"{Name}\r\n{Type}\r\n{Value}");
+        }
+
+        private void CopyToClipboardEntryWithPath()
+        {
+            ClipboardHelper.CopyToClipboardAction($"{Key}\r\n{Name}\r\n{Type}\r\n{Value}");
```

Comment:
Wondering if it is better to copy with a blank line between the information instead of a simple line break? 🤔